lnrpc/gen_protos.sh

```diff
@@ -6,19 +6,19 @@ echo "Generating root gRPC server protos"
 protoc -I/usr/local/include -I. \
        -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
        --go_out=plugins=grpc,paths=source_relative:. \
-       rpc.proto
```

Comment:
note: this may break people's scripts if they are assuming them all to be in one file

Reply:
Oh, that's right. This will be a breaking change for JS users that load the file every time the application starts. Should be fine for anyone that uses some compiled form of protos as they will still remain valid (but probably break after they compile next and don't do so for the unlocker proto). Not sure if we need to discuss this some more offline, get more opinions on it?

Reply:
yeah probably not a huge issue, this is arguably how it should have been from the get  go.